docs/testspec/src/12_pubkey_kem.rst

```diff
@@ -1,6 +1,124 @@
+size of the other values, all values except the KAT seeds and the ciphertexts
+are hashed.
+
+.. table::
+   :class: longtable
+   :widths: 20 80
+
+   +------------------------+-------------------------------------------------------------------------+
+   | **Test Case No.:**     | PKENC-CMCE-1                                                            |
+   +========================+=========================================================================+
+   | **Type:**              | Known Answer Tests                                                      |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Description:**       | Uses the KAT vectors of Classic McElieces's reference implementation as |
+   |                        | specified in the NIST submission                                        |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Preconditions:**     | None                                                                    |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Input Values:**      | Test Vectors with RNG seed inputs in:                                   |
+   |                        |                                                                         |
+   |                        | * :srcref:`src/tests/data/pubkey/cmce_kat_hashed.vec`                   |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Expected Output:**   | Above described test vector files contain expected values for:          |
+   |                        |                                                                         |
+   |                        | * Classic McEliece Public Key                                           |
+   |                        | * Classic McEliece Private Key                                          |
+   |                        | * Ciphertext                                                            |
+   |                        | * Shared Secret                                                         |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Steps:**             | For each KAT vector:                                                    |
+   |                        |                                                                         |
+   |                        | #. Seed a AES-256-CTR-DRBG with the specified RNG seed                  |
+   |                        |                                                                         |
+   |                        | #. Use the seeded RNG to generate a Classic McEliece key pair and       |
+   |                        |    compare it to the expected public and private key in the test        |
```

Comment:
```suggestion
   |                        |    compare its private key to the expected public and private key in the test        |
```

Reply:
Both keys are compared with the KAT data.